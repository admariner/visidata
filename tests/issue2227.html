<table>
 <tr><th>url</th></tr>
 <tr><td><a href="https://google.com]">1</a></td></tr>
 <tr><td><a href="https://example.com">2</a></td></tr>
</table>
