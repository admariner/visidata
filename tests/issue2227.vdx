# VisiData v3.0
open-file tests/issue2227.html
open-row
