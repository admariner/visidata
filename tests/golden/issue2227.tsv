url	url_link0
1	https://google.com]
2	https://example.com
